api/src/main/resources/messages.properties

```diff
@@ -17,4 +17,6 @@ ${project.parent.artifactId}.settings=Settings
 ${project.parent.artifactId}.saved=Global Properties Saved
 ${project.parent.artifactId}.appBaseUrl.label = App Base URL
 ${project.parent.artifactId}.appFolderPath.label = App Folder Path
-${project.parent.artifactId}.appStoreUrl.label = App Store URL
\ No newline at end of file
+${project.parent.artifactId}.appStoreUrl.label = App Store URL
+${project.parent.artifactId}.invalid_owa_Url = The OWA installation URL passed is invalid. The URL should lead to a downloadable OWA zip file
+${project.parent.artifactId}.missing_owa_file = Installation URL doesnot specify download file
```

Comment:
Whats is "doesnot"?

Reply:
Again, is the end user specifying a URL or File?

Reply:
Changed it to reflect a file